### PR TITLE
New nearline monitoring 

### PR DESCRIPTION
**Pull request by tannerbk**
Introduce new nearline monitoring tab that now includes:

- Monitoring of channel error flags
- Monitoring of trigger clock jumps
- Monitoring of occupancy by trigger type
- summary of all nearline monitoring jobs, including ping crates
- updates to nearline summary page to include run type

This is not quite finished, in particular the occupancy page, but its pretty close and might take a little while to review.

### CODE REVIEW

**Comment by tannerbk** [comment]
OK I think this should be good to go, although its a fairly large amount of code, so I'll continue to look for bugs. There are some changes to the layout hat you might not like, so let me know if you have suggestions. Otherwise most of the changes are related to the new "nearline monitoring" pages.

**Comment by tlatorre-uchicago** [comment]
Should the occupancy by trigger page be showing anything yet?

**Comment by tannerbk** [comment]
I think so, there are only two runs but you should see them if you increase the limit.

File: minard/nearline_settings.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # List of all nearline job types
-jobTypes = ["All","DAQ","SMELLIE","BLINDNESS_CHUNKER","PCA_TELLIE","RUN","TELLIE","ECA","PING_CRATES","PMTCalStatus","ANALYSIS","DQHL","CHS","DQLL","PMTNOISE","CHANNEL_FLAGS","CLOCK_JUMPS"]
+jobTypes = ["All","DAQ","SMELLIE","BLINDNESS_CHUNKER","PCA_TELLIE","RUN","TELLIE","ECA","PING_CRATES","PMTCalStatus","ANALYSIS","DQHL","CHS","DQLL","PMTNOISE","CHANNEL_FLAGS","CLOCK_JUMPS", "TRIGGER_OCCUPNACY"]
```
Comment (tlatorre-uchicago) [inline minard/nearline_settings.py]: typo?

File: minard/occupancy.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                          (trigger_type, run))
+
+    rows = result.fetchall()
+    if not rows:
+        return None
+
+    data = [0]*9728
+    trigger_norm = 0
+    for lcn, norm, occupancy in rows:
+        trigger_norm = norm
```
Comment (tlatorre-uchicago) [inline minard/occupancy.py]: Just curious, is this supposed to just be reset every time the loop runs?

Comment (tannerbk) [reply minard/occupancy.py]: Yah this is just the same number over the loop so its a bit silly

**Comment by tannerbk** [comment]
I initially added the check rates results to the summary page but it ended up being quite slow

**Pull request merged**